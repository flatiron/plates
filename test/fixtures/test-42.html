<div>
  <div class="page-details">
    <div class="author">
      <div class="name"></div>
    </div>
  </div>
  <div class="page-details">
    <h1 class="title"></h1>
  </div>
</div>
