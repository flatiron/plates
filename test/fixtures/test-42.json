{
  "page-details": {
    "author": {
      "name": "Maciej"
    },
    "title": "Plates"
  }
}
