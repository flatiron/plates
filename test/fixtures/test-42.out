<div>
  <div class="page-details">
    <div class="author">
      <div class="name">Maciej</div>
    </div>
  </div>
  <div class="page-details">
    <h1 class="title">Plates</h1>
  </div>
</div>
